app-blank-after-kendo-upgrade.md

```diff
@@ -0,0 +1,43 @@
+
+---
+title: Upgrading from Kendo UI Core to Professional Breaks App
+description: After upgrading an app through the Package Manager to use Kendo UI Professional it appears blank on start up. 
+type: troubleshooting
+page_title: Upgrading from Kendo UI Core to Professional Breaks App
```

Comment:
If you want to use the same title, there's a logic that if page_title is not specified, Jekyll generates the page title based on the title.